src/main/java/io/github/guacsec/trustifyda/integration/backend/ExhortIntegration.java

```diff
@@ -268,12 +268,7 @@ public void configure() {
         .parallelProcessing()
         .process(this::setProviderConfiguration)
         .setBody(exchangeProperty("originalBody"))
-        .choice()
```

Comment:
if you look at the previous part, it splits and aggregates the response for osv and trustify, we don't need that anymore